src/main/java/htsjdk/samtools/util/zip/DeflaterFactory.java

```diff
@@ -43,36 +43,12 @@
  */
 public class DeflaterFactory {
 
-    private static Constructor<IntelDeflater> intelDeflaterConstructor;
```

Comment:
Javadoc for this class needs updating to reflect the changes that have been made.  Maybe just change it to read something like: `Constructs deflaters as needed by the library. Simply constructs a default JDK Deflater but provides a central point for changing the behavior in future.`